utils/rss-feed.ts

```diff
@@ -0,0 +1,32 @@
+import Parser from 'rss-parser';
+import dotenv from 'dotenv';
+import express from 'express';
+import publicationsJSON from '../publications.json';
+
+const PORT = 5000;
+
+// load the environment variables from the .env file
+dotenv.config({
+    path: '.env',
+});
+
+const publicationsDB = publicationsJSON.publications;
+const FEED_LIST = publicationsDB.map((publication) => {
+    return publication.feed;
+  });
+
+const app = express();
+
+app.get('/', (req, res) => {
+    const parser = new Parser();
+
+    const feedRequests = FEED_LIST.map((feed) => {
+        return parser.parseURL(feed);
+    });
+
+    Promise.all(feedRequests).then((response) => {
+        res.setHeader('Access-Control-Allow-Origin', '*');
+        res.header('Access-Control-Allow-Methods', 'GET');
+        res.json(response);
+    });
```

Comment:
Not too familiar with `async-await` syntax vs. Promises but I do know the former is a newer feature. Is there a reason we're choosing to use `.then()` syntax instead of `await`? Also what happens if one of the feed requests goes wrong (we don't have any `.catch()` defined)?

Reply:
I didn’t spend too much time with this code, this was just a starter file rss-parser had on their docs site and modified to see all of the articles in one feed. I could change it here if it’s considered better practice (and a few other places .then is used)

Reply:
+1 to this!

Reply:
`Promise.all` is fine here imo; in the case that any of the feed requests fail, we get a fast fail which we wouldn't with multiple awaits